chess-AI/train.py

```diff
@@ -1,10 +1,18 @@
 import random
 
 import chess
+import torch
+import numpy as np
+import time
+from torch.autograd import backward
+from torch.functional import Tensor
+from torch.utils.data import DataLoader, TensorDataset
```

Comment:
Same comment as above, either selectively import from torch for everything, or just import torch and then do `torch.*` as you need